pkg/importer/vddk-datasource_amd64.go

```diff
@@ -554,7 +554,7 @@ func GetBlockStatus(handle NbdOperations, extent types.DiskChangeExtent) []*Bloc
 					// Merge with previous block
 					blocks[last] = &BlockStatusData{
 						Offset: lastBlock.Offset,
-						Length: lastBlock.Length,
+						Length: lastBlock.Length + length,
```

Comment:
This seems like the meat of the change, looks good.

Reply:
@mrnold looks like `BlockStatusData.Length` is a uint32.  Do we have to be concerned about adding an int64 to that?

Reply:
okay so length initially comes in as a uint32 but don't we still have to be concerned about overflow?

Reply:
Yes, overflow is a concern, so this pull request changes BlockStatusData.Length to be an int64 in commit 27e07d36a87c52611c1fef4e6a9f5ceb2814c3bf.

Reply:
Ah okay missed that